packages/components/src/Chips/InternalChipSingleSelect.tsx

```diff
@@ -34,7 +35,15 @@ export function InternalChipSingleSelect({
               }}
               disabled={child.props.disabled}
             />
-            <InternalChip {...child.props} active={isSelected} />
+            <InternalChip
+              {...child.props}
+              {...(isSelected
+                ? {
+                    suffix: <Icon size="small" name="checkmark" />,
```

Comment:
Need to spec `color="interactiveSubtle` here otherwise we get the default "success" green: 
![image](https://github.com/user-attachments/assets/3213bf8c-1275-457f-bf55-87f9c6ea62be)

Reply:
these were both quick fixes. pushed up em.